### ISSUE TITLE
Allow input file in NCCH format (decrypted .cxi as extractable from ninfs)

### ISSUE THREAD

**Issue by geekley** [rickfay/z17-randomizer#16]
I assume the code only uses data from the NCCH.
Would it be possible to also allow a decrypted .cxi file as input (as extracted from the .cci/.3ds by a tool like [ninfs](https://github.com/ihaveamac/ninfs))?

This would allow to save more than 300 MiB of space by keeping just the decrypted .cxi (NCCH) file. I'm asking because maybe it's simple to implement, like using just a different offset in the file or something, or maybe you're already extracting it and just need to skip this step in this case.

Also, since this is command-line only, can you provide a Linux executable too?

**Comment by rickfay** [comment]
I'm happy to share that the newest Dev Build does offer a Linux executable!

**Comment by rickfay** [comment]
I can look into allowing a decrypted .cxi, it's not the highest priority but if it's not too complicated I'll see what we can do.

**Comment by benny-dreamly** [comment]
yeah, allowing a .cci file would be nice for azahar not liking 3ds files for _some_ reason
